### PR TITLE
Add API functions

### PR DESCRIPTION
**Pull request by DiegoMero**
This pull request is to add the API functions to send and get data from the microverse Leaderboard API service.

### CODE REVIEW

File: src/modules/addScore.js [hunk 1]
```diff
@@ -1,6 +1,4 @@
-import getScore from './getScore.js';
-
-export function addScore(score) {
+export default function addScore(score) {
```
Comment (VICTOR-CHUKWU) [inline src/modules/addScore.js]: - [ ] Kindly change this to an arrow function

File: src/modules/displayScoreList.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import getScore from './getScore.js';
-import { addScore } from './addScore.js';
+import addScore from './addScore.js';
 
 export default function displayScoreList() {
```
Comment (VICTOR-CHUKWU) [inline src/modules/displayScoreList.js]: - [ ] Kindly change this to an arrow function

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @DiegoMero  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @DiegoMero, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Clean and organized code. 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**